src/lib/workspace/SessionPanel.tsx

```diff
@@ -57,13 +69,22 @@ export function SessionPanel(props: Props) {
               deleteSession={deleteSession}
             />
             <Route
-              path={[`${routeBase}/variables/:entityId?`, `${routeBase}`]}
-              render={(props: RouteComponentProps<{ entityId?: string }>) => (
+              path={[
+                `${routeBase}/variables/:entityId?/:variableId?`,
+                `${routeBase}`,
```

Comment:
Second element could be just `routeBase` - but there is also some nice symmetry to the way it is written now.

Reply:
I decided to just leave it. It's doesn't really hurt much :slightly_smiling_face: